Src/Web/Web.Shared.Net/RequestTrackingTelemetryModule.cs

```diff
@@ -267,6 +267,13 @@ public void Initialize(TelemetryConfiguration configuration)
             this.telemetryClient = new TelemetryClient(configuration);
             this.telemetryClient.Context.GetInternalContext().SdkVersion = SdkVersionUtils.GetSdkVersion("web:");
 
+            this.DisableTrackingProperties = configuration.EvaluateExperimentalFeature("deferRequestTrackingProperties");
```

Comment:
> internal bool DisableTrackingProperties { get; set; } = false;

shouldn't they become private now?

Reply:
I think it had some tests that toggled this on/off.
tests require `internal`